backend/app/pydantic_models/notes.py

```
@@ -1 +1,11 @@
 # do something here bruh
+from pydantic import BaseModel
+from typing import List
+
+class Note(BaseModel):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)